### DISCUSSION TITLE
2022-06-13-a-good-markdown

### DISCUSSION THREAD

**Discussion by tw93** [tw93/tw93.github.io#105] [General]
https://tw93.fun/2022-06-13/a-good-markdown-copy.html 

 Hi，我是 Tw93，是一个前端工程师，空闲也折腾 iOS / macOS ，喜欢开源分享交流。Tool最近将自己两年前做的一个 macOS 的 Markdown 笔记应用给完善了一下，放出来给大伙试试。

**Comment by maajiko** [comment]
有点喜欢这个官网怎么办，可以白嫖吗付费也可以🥰

**Comment by tw93** [comment]
> @maajiko
> 有点喜欢这个官网怎么办，可以白嫖吗付费也可以🥰
哈哈 你可以去喂猫，给我家汤圆可乐买罐头

**Comment by muzichen** [comment]
有点好奇，你在写这个工具的时候，会先去参考别人的实现方法么，然后再自己动手去写？

**Comment by tw93** [comment]
> @muzichen
> 有点好奇，你在写这个工具的时候，会先去参考别人的实现方法么，然后再自己动手去写？ 

会的，会去看很多笔记应用的用户使用，挑选里面我认为不错的，或者我很喜欢的

**Reply by muzichen** [reply]
代码呢，会参考么，还是自己去实现。

**Reply by tw93** [reply]
会参考哈，毕竟我只是一个 swift 爱好者

**Comment by xwzbupt** [comment]
这个编辑器只能支持MacOS吗？

**Reply by tw93** [reply]
是的 后面会有机会支持 iPhone

**Comment by huyixi** [comment]
现在能否支持 打开一个已存在的文件夹（类似 vscode 的工作区） 的功能

**Comment by yo1995** [comment]
看到这么漂亮的landing page，闭着眼睛按零，打钱就完事儿了！ 😎
